src/GitHub.UI/Converters/TrimNewlinesConverter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace GitHub.UI
+{
+    /// <summary>
+    /// An <see cref="IValueConverter"/> that trims newlines from a string and replaces them
+    /// with spaces.
+    /// </summary>
+    public class TrimNewlinesConverter : ValueConverterMarkupExtension<TrimNewlinesConverter>
+    {
+        public override object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var text = value as string;
+            if (String.IsNullOrEmpty(text)) return null;
+            return Regex.Replace(text, @"\t|\n|\r", " ");
```

Comment:
If we're displaying a comment on a single line, it would make sense to me to convert tabs to spaces, because otherwise we'd be wasting space. Tabs are generally used for alignment between lines, so if we've got no lines it makes sense to strip them. Do you have an example where this wouldn't be a good idea?

Reply:
Oh, I was just comparing it to the comment: "trims newlines from a string and replaces them with spaces".